pdi-dataservice-client/src/main/java/org/pentaho/di/core/jdbc/ThinUtil.java

```diff
@@ -655,4 +655,38 @@ public static String resolveFieldName( String field, RowMetaInterface serviceFie
     }
     return Optional.ofNullable( valueMeta );
   }
+
+  /**
+  * Remove first and last quote if string contains double quote
+  * */
+  public static String unQuote( String str ) {
+    StringBuilder builder = new StringBuilder( str );
+    if ( builder.indexOf( "\"\"" ) > 0 &&  builder.length() > 0 && builder.charAt( 0 ) == '"'
+      && builder.charAt( builder.length() - 1 ) == '"' ) {
+      builder.deleteCharAt( builder.length() - 1 );
+      builder.deleteCharAt( 0 );
+    }
+    return builder.toString();
+  }
+
+  /**
+   * Replaces each substring of this string that matches the given regular expression with the given replacement.
+   * @param strSrc source string
+   * @param strA expression to which this string is to be matched
+   * @param replacement the string to be substituted for each match
+   *
+   * */
+  public static String replaceSubString( String strSrc, String strA, String replacement ) {
```

Comment:
So why do this when String has a replaceAll method that accepts a regular expression ?
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replaceAll(java.lang.String,%20java.lang.String)